stdlib/3/subprocess.pyi

```diff
@@ -8,14 +8,14 @@ from types import TracebackType
 
 if sys.version_info >= (3, 5):
     class CompletedProcess:
-        args = ...  # type: Union[List, str]
+        args = ...  # type: Union[Sequence[str], str]
         returncode = ...  # type: int
-        stdout = ...  # type: Any
-        stderr = ...  # type: Any
+        stdout = ...  # type: Union[str, bytes, None]
+        stderr = ...  # type: Union[str, bytes, None]
```

Comment:
I'm worried that making this into a Union where it really depends on an argument passed to run(), will break correct code. This is the same problem as occurs for e.g. open(), and it's why the latter is defined as `-> IO[Any]`.